src/main/java/com/leahtcodes/Controllers/UserController.java

```diff
@@ -29,15 +59,19 @@ public Optional<Userss> findUserById(@RequestParam(value = "id",required = true)
         return userRepository.findById(id);
 
     }
-    @GetMapping("/coffees/")
-    public List<Coffee> getAllCoffees(@RequestParam(value = "id", required = true) Long id){
-        Optional<Userss> user = userRepository.findById(id);
-        if(!user.isPresent()){
-            return new ArrayList<Coffee>();
-        }else{
-            return user.get().getCoffees();
-        }
+    @GetMapping("/coffees")
+    public List<Coffee> getAllCoffees( OAuth2AuthenticationToken authentication ){
+        OAuth2AuthorizedClient client = authorizedClientService
+                .loadAuthorizedClient(
+                        authentication.getAuthorizedClientRegistrationId(),
+                        authentication.getName());
+
+        Map<String, Object> attributes = authentication.getPrincipal().getAttributes();
+       String googleId = (String) attributes.get("sub");
+        Userss user = userRepository.findByGoogleId(googleId);
+            return user.getCoffees();
     }
+
     @DeleteMapping("")
     public Userss deleteUser(@RequestParam(value="id", required = true) Long id){
```

Comment:
You definitely want to protect this more.  Make sure the user is valid, logged in, etc.  Right now, unless I'm missing something, they can just remove someone